src/ciphersuite/codec.rs

```diff
@@ -84,3 +84,15 @@ impl Codec for ReuseGuard {
         Ok(ReuseGuard { value: guard })
     }
 }
+
+impl Codec for Secret {
```

Comment:
I think we _do_ send secrets over the wire, for example when sending a path secrets to the co-path the course of an update.

Reply:
That's probably a good idea. Quite a few Secrets already have their own wrapper struct such as `PathSecret`, but then we'd need to create a few more wrapper structs, this is partially because all Secrets in `EpochSecrets` need to be encoded (although if can get rid of `Codec` for `MlsGroup`, that would disappear).

Also, I think those Secret wrappers would have to live in `ciphersuites/mod.rs` so they can access Secret.value directly.